src/components/create-safe/InfoWidget/index.tsx

```diff
@@ -1,84 +1,92 @@
 import type { AlertColor } from '@mui/material'
 import type { ReactElement } from 'react'
-
 import LightbulbIcon from '@/public/images/common/lightbulb.svg'
-
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
 import css from './styles.module.css'
 
-type Props = {
+type InfoWidgetProps = {
   title: string
   steps: { title: string; text: string }[]
   variant: AlertColor
+  startExpanded?: boolean
 }
 
-const InfoWidget = ({ title, steps, variant }: Props): ReactElement | null => {
-  const [activeStep, setActiveStep] = useState(0)
-  const [dismissed, setDismissed] = useState(false)
-
-  const isFirst = activeStep === 0
-  const isLast = activeStep === steps.length - 1
-
+const InfoWidget = ({ title, steps, variant, startExpanded = false }: InfoWidgetProps): ReactElement | null => {
   const isMultiStep = steps.length > 1
 
-  const onPrev = () => {
-    if (!isFirst) {
-      setActiveStep((prev) => prev - 1)
-    }
-  }
-
-  const onNext = () => {
-    if (isLast) {
-      setDismissed(true)
-    } else {
-      setActiveStep((prev) => prev + 1)
-    }
-  }
-
-  if (dismissed) {
+  if (steps.length === 0) {
     return null
   }
 
   return (
     <Card sx={{ backgroundColor: ({ palette }) => palette[variant]?.background }}>
       <CardHeader
-        className={css.header}
+        sx={{ paddingBottom: '0px' }}
         title={
-          <Box className={css.headerWrapper}>
-            <Typography
-              variant="caption"
-              className={css.title}
-              sx={{
-                backgroundColor: ({ palette }) => palette[variant]?.main,
-              }}
-            >
-              <SvgIcon component={LightbulbIcon} inheritViewBox fontSize="inherit" className={css.lightbulb} />
-              {title}
-            </Typography>
-            {isMultiStep && (
+          <>
+            <Box className={css.headerWrapper}>
+              <Typography
+                variant="caption"
+                className={css.title}
+                sx={{
+                  backgroundColor: ({ palette }) => palette[variant]?.main,
+                }}
+              >
+                <SvgIcon component={LightbulbIcon} inheritViewBox fontSize="inherit" className={css.lightbulb} />
+                {title}
+              </Typography>
               <Typography variant="caption" className={css.count}>
-                {activeStep + 1} of {steps.length}
+                {steps.length} tip{isMultiStep && 's'}
               </Typography>
-            )}
-          </Box>
+            </Box>
+          </>
         }
       />
-      <CardContent>
-        <Typography variant="h5">{steps[activeStep].title}</Typography>
-        <Typography variant="body2">{steps[activeStep].text}</Typography>
+      <CardContent sx={{ padding: '0', '&.MuiCardContent-root': { paddingBottom: '0px' } }}>
+        {steps.map(({ title, text }) => {
+          return (
+            <Accordion
+              key={title.replace(' ', '-')}
```

Comment:
Was React complaining about the spaces?

Reply:
No. Do you discourage this replacement?

Reply:
I'd suggest we remove it if there were no warnings. It's excessive and the `key` is only used internally.

Reply:
I've removed it